moda-contracts/src/interfaces/IReleases.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+/**
+ * @title IReleases
+ * @dev Interface for Releases
+ */
+interface IReleases {
+    event ReleaseCreated(uint256 tokenId);
+    event ReleaseWithdrawn(address indexed receiver, uint256 tokenId, uint256 amount);
+
+    // /**
+    //  * @dev Initializes the contract
+    //  * @param admin The address that will be given the role of default admin. See {AccessControl}
+    //  * @param releaseAdmins An array of addresses that will be given the role of release admin
+    //  * @param name_ The name of the Releases contract
+    //  * @param symbol_ The symbol of the Releases contract
+    //  * @param registry The address of the registry contract
+    //  * @param modaSplits The address of the moda splits contract
+    //  */
+    // function initialize(
```

Comment:
Let's remove all the dead code / commented out code if we aren't using it in this PR